### PR TITLE
Rename parent & user_in_role built-in RBAC predicates

### PR DESCRIPTION
**Pull request by leina05**
PR checklist:

- [ ] Added changelog entry.
- [ ] Open PR in GitClub repo to reflect these changes and confirm docs are correct

### CODE REVIEW

File: docs/content/python/new-roles/examples.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ In this guide, we'll cover a few different role models in the context of
 the [GitClub example application](https://github.com/osohq/gitclub-sqlalchemy-flask-react).
 
 {{% callout "Read first" "blue" %}}
- - [Getting started](getting-started)
-{{% /callout %}}
+
+- [Getting started](getting-started)
+  {{% /callout %}}
```
Comment (gj) [inline docs/content/python/new-roles/examples.md:16]: autocorrect run amok

File: docs/content/python/new-roles/examples.md [hunk 2]
```diff
@@ -27,12 +28,13 @@ To define a role for another resource type, we add another `resource`
...
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
->}}
+path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
+from="docs: begin-repo-resource"
+to="docs: end-repo-resource"
+gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
+linenos=true
+
+> }}
```
Comment (gj) [inline docs/content/python/new-roles/examples.md:37]: is this valid after going through the autocorrect thresher?

File: docs/content/python/new-roles/examples.md [hunk 3]
```diff
@@ -53,28 +55,30 @@ roles**.
...
-    linenos=true
->}}
+path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
+from="docs: begin-repo-parent"
+to="docs: end-repo-parent"
+gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
+linenos=true
 
-This `parent(child, parent)` rule defines that an organization is
+> }}
```
Comment (gj) [inline docs/content/python/new-roles/examples.md:64]: pls turn off the autocorrecter so it's easier to see what was actually changed 🙏

Comment (leina05) [reply docs/content/python/new-roles/examples.md:64]: I know I always forget but there's no way to automatically turn it off for just one language

File: docs/content/any/getting-started/policies/index.md [hunk 4]
```diff
@@ -516,9 +516,9 @@ whatever you like, and use them from within your `allow` rules.
 For instance a rule that handles roles might look like this:
 
 ```polar
-allow(user, action, resource) if
+allow(actor, action, resource) if
     resource_role_applies_to(resource, role_resource) and
-    user_in_role(user, role, role_resource) and
+     actor_can_assume_role(actor, role, role_resource) and
```
Comment (leina05) [inline docs/content/any/getting-started/policies/index.md:521]: @gj legit change

File: docs/content/any/learn/roles/index.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ an [allow rule](glossary#allow-rules) that calls these rules looks like this:
 
 ```polar
 allow(actor, action, resource) if
-    user_in_role(actor, resource, role) and
+     actor_can_assume_role(actor, resource, role) and
```
Comment (leina05) [inline docs/content/any/learn/roles/index.md:35]: @gj legit change

File: docs/content/python/new-roles/examples.md (hunk 2 above)
Comment (leina05) [reply docs/content/python/new-roles/examples.md:37]: ugh let me just rebase

**Review by gj — COMMENTED** [review]
Looks like this PR includes `child_parent/2` & `actor_can_assume_role/3`, but both changes look good.

Would you mind adding the changelog entry for both changes (for the SQLAlchemy roles lib)? Getting worried about how many changelogs entries we're procrastinating on (mostly mine), and this one is distinct from the main entry I still need to write since this pertains to SQLAlchemy roles instead of Polar roles.

**Comment by leina05** [comment]
> Looks like this PR includes `child_parent/2` & `actor_can_assume_role/3`, but both changes look good.
> 
> Would you mind adding the changelog entry for both changes (for the SQLAlchemy roles lib)? Getting worried about how many changelogs entries we're procrastinating on (mostly mine), and this one is distinct from the main entry I still need to write since this pertains to SQLAlchemy roles instead of Polar roles.

I'm also adding the `role_allows` change in here since it's so small. that doesn't require a changelog entry since it's part of the new Polar roles stuff (just needs to be correct in that main entry). I can do the sqlalchemy changelog entries.

**Review by gj — APPROVED** [review]
can fix the changelog in a follow-up

**Pull request merged**